tests/integration/buildcmd/build_integ_base.py

```diff
@@ -120,10 +120,23 @@ def verify_docker_container_cleanedup(self, runtime):
             time.sleep(1)
         docker_client = docker.from_env()
         samcli_containers = docker_client.containers.list(
-            all=True, filters={"ancestor": "lambci/lambda:build-{}".format(runtime)}
+            all=True, filters={"ancestor": f"public.ecr.aws/sam/build-{runtime}"}
         )
         self.assertFalse(bool(samcli_containers), "Build containers have not been removed")
 
+    def verify_pulling_only_latest_tag(self, runtime):
```

Comment:
nit: this doesn't actually check that `latest` was pulled, just that _an_ image was pulled

Reply:
Thanks for pointing it out! Added a more explicit check and will be included in a future PR.